### ISSUE TITLE
[BUG]: Managed User URL Redirect

### ISSUE THREAD

**Issue by kartikmehta8** [activepieces/activepieces#3571]
**Describe the bug**
If you vision this [link](https://www.activepieces.com/docs/embedding/embed-builder/embed-iframe) and,
1. Under this [heading](https://www.activepieces.com/docs/embedding/embed-builder/embed-iframe#step-2-generate-a-jwt-token), the [managed user URL](https://www.activepieces.com/docs/platform/managed-users) is redirecting to the main page of the document.
2. Under [Step: 3](https://www.activepieces.com/docs/embedding/embed-builder/embed-iframe#step-3-add-an-iframe), the hyperlink of **[this guide](https://www.activepieces.com/docs/platform/managed-users)** has the same problem.

Basically, I am not able to visit this URL: [www.activepieces.com/docs/platform/managed-users](https://www.activepieces.com/docs/platform/managed-users)

**Expected behavior**
[www.activepieces.com/docs/platform/managed-users](https://www.activepieces.com/docs/platform/managed-users) should be found without any redirection to the home page, if I am not wrong.

Thank you!

**Comment by abuaboud** [comment]
Thank you, It should be fixed, It should redirect you to this page

[activepieces.com/docs/embedding/embed-builder/generate-jwt](https://www.activepieces.com/docs/embedding/embed-builder/generate-jwt)
